cg_hermes/cli/export.py

```diff
@@ -75,7 +75,7 @@ class OutputFormat(StrEnum):
 @app.command(name="tags")
 def export_tags_cmd(
     output: OutputFormat = typer.Option(OutputFormat.github),
-    pipeline: Pipeline = None,
+    pipeline: Workflow = None,
```

Comment:
isn't this str?

Reply:
This way should lock it to the defined strings in the class. But I will add | None.

Reply:
Turn out Typer currently cannot handle the None syntax - leaving it